lib/services/local/access_list.go

```diff
@@ -1045,6 +1036,18 @@ func (a *AccessListService) VerifyAccessListCreateLimit(ctx context.Context, tar
 	return trace.AccessDenied("%s", limitReachedMessage)
 }
 
+func preserveAccessListFields(existingAccessList, accessList *accesslist.AccessList) {
+	if existingAccessList != nil {
+		// Set MemberOf/OwnerOf to the existing values to prevent them from being updated.
+		accessList.Status.MemberOf = existingAccessList.Status.MemberOf
+		accessList.Status.OwnerOf = existingAccessList.Status.OwnerOf
+	} else {
+		// For newly created AccessList make sure MemberOf/OwnerOf are empty.
+		accessList.Status.MemberOf = []string{}
+		accessList.Status.OwnerOf = []string{}
+	}
+}
```

Comment:
https://go.dev/wiki/CodeReviewComments#indent-error-flow:~:text=else%20%7B%0A%20%20%20%20//%20normal%20code%0A%7D-,Instead%2C%20write%3A,-if%20err%20!%3D%20nil

```suggestion
func preserveAccessListFields(existingAccessList, accessList *accesslist.AccessList) {
	if existingAccessList != nil {
		// Set MemberOf/OwnerOf to the existing values to prevent them from being updated.
		accessList.Status.MemberOf = existingAccessList.Status.MemberOf
		accessList.Status.OwnerOf = existingAccessList.Status.OwnerOf
		return
	}

      // For newly created AccessList make sure MemberOf/OwnerOf are empty.
      accessList.Status.MemberOf = []string{}
      accessList.Status.OwnerOf = []string{}
}
```

Reply:
This page talks about normal code flow. In this function, both branches are a normal flow and I somewhat like the visual alignment.